packages/syft/src/syft/client/registry.py

```diff
@@ -22,20 +22,39 @@
 NETWORK_REGISTRY_URL = (
     "https://raw.githubusercontent.com/OpenMined/NetworkRegistry/main/gateways.json"
 )
+
 NETWORK_REGISTRY_REPO = "https://github.com/OpenMined/NetworkRegistry"
 
 
+def _get_all_networks(network_json: dict, version: str) -> list[dict]:
+    return network_json.get(version, {}).get("gateways", [])
+
+
 class NetworkRegistry:
     def __init__(self) -> None:
         self.all_networks: list[dict] = []
+
+        try:
+            network_json = self.load_network_registry_json()
+            self.all_networks = _get_all_networks(
+                network_json=network_json, version="2.0.0"
+            )
+        except Exception as e:
+            warning(
+                f"Failed to get Network Registry, go checkout: {NETWORK_REGISTRY_REPO}. {e}"
```

Comment:
nitpick
```suggestion
                f"Failed to get Network Registry from {NETWORK_REGISTRY_REPO}. {e}"
```

only state the problem here, don't need to specify action "go checkout..."

Reply:
Fixed. Thanks @kiendang